config/configuration.yaml.example

```diff
@@ -102,3 +102,15 @@ automation 2:
 
   execute_service: notify.notify
   service_data: {"message":"It's 4, time for beer!"}
+
+sensor:
+  platform: systemmonitor
+  devices:
```

Comment:
Could you pick a better name? It's not devices that you're monitoring. What about resources?